metricbeat/module/prometheus/collector/collector.go

```diff
@@ -24,9 +24,10 @@ var (
 )
 
 func init() {
-	if err := mb.Registry.AddMetricSet("prometheus", "collector", New, hostParser); err != nil {
-		panic(err)
-	}
+	mb.Registry.MustAddMetricSet("prometheus", "collector", New,
```

Comment:
We should have a mention of this in the docs.

@andrewkroh Thanks for this feature. We should probably make use of it in some other modules.